catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -269,15 +248,17 @@ private QueryResponse sourceFederate(List<Source> sources, final QueryRequest qu
                         logger.warn("Plugin stopped processing", e);
                     }
 
-                    finalQueryRequest = modifiedQueryRequest;
                     if (source instanceof CatalogProvider && SystemInfo.getSiteName()
                             .equals(source.getId())) {
-                        finalQueryRequest = getQueryRequestWithAdditionalFilters(
-                                modifiedQueryRequest);
+                        sourceQueryRequest =
```

Comment:
I think this if statement should go away completely and the validationQueryFactory (which is just adding tags to the query) should be put in the new pre-federation plugin.